openapi.yaml

```diff
@@ -686,6 +686,7 @@ components:
       type: object
 
     Error:
+      description: "See [Service Broker Errors](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#service-broker-errors) for more details."
```

Comment:
too bad we can't make this a relative link so we'd have referential integrity within each release.